nl_server/embeddings_map.py

```diff
@@ -16,21 +16,23 @@
 import logging
 import os
 import shutil
-from typing import List
+from typing import Dict, List
 
 import pandas as pd
 
 from nl_server.config import CUSTOM_DC_INDEX
 from nl_server.config import DEFAULT_INDEX_TYPE
 from nl_server.config import EmbeddingsIndex
-from nl_server.embeddings import Embeddings
-from nl_server.embeddings import load_model
+from nl_server.model.sentence_transformer import SentenceTransformerModel
+from nl_server.store.memory import MemoryEmbeddingsStore
+from nl_server.wrapper import Embeddings
+from nl_server.wrapper import EmbeddingsModel
 
 
 #
-# A simple wrapper class around multiple embeddings indexes.
+# A map from specific index to embeddings stores and models.
 #
-class Store:
+class EmbeddingsMap:
 
   def __init__(self, indexes: List[EmbeddingsIndex]):
```

Comment:
here assumes relation between index and models. I guess all of those should eventually be config driven.